app/react/utils/api.js

```diff
@@ -20,15 +20,58 @@ const doneLoading = data => {
 
 const isNonUsualApiError = error => error.status && ![401, 404, 409, 500].includes(error.status);
 
+const errorMessages = [
+  {
+    key: /PayloadTooLargeError: request entity too large/g,
+    message: 'The request has too large data. Please review any long value property.',
+  },
+  {
+    key: /max_bytes_length_exceeded_exception. Invalid Fields: (.+)/g,
+    message: 'The request has too large data. Please review the follow fields: {0} ',
```

Comment:
“Follow” is a verb “to go after someone”.  In this case you want them to review “the following fields”.   So:

“Request data too large. Please review the following fields: “

Reply:
this code was removed because it became unreachable after adding the ajv validation